aries-site/src/tests/accessibility/axe.js

```diff
@@ -23,7 +23,12 @@ test('should check Home page', async t => {
     const axeContext = {
       exclude: [],
     };
-    const axeOptions = { rules: { 'color-contrast': { enabled: false } } };
+    const axeOptions = {
+      rules: {
+        'color-contrast': { enabled: false },
```

Comment:
should we add a comment to enable it once the colors are approved?

Reply:
Yes, I'll add one into the file.

Reply:
Done.